### PR TITLE
Update javascript.md

### PR DESCRIPTION
**Pull request by XhmikosR**

### CODE REVIEW

File: site/content/docs/5.2/getting-started/javascript.md [hunk 1]
```diff
@@ -302,11 +301,17 @@ const tooltip = new bootstrap.Tooltip(yourTooltipEl, {
 **You don't need jQuery in Bootstrap 5**, but it's still possible to use our components with jQuery. If Bootstrap detects `jQuery` in the `window` object, it'll add all of our components in jQuery's plugin system. This allows you to do the following:
 
 ```js
-$('[data-bs-toggle="tooltip"]').tooltip() // to enable tooltips, with default configuration
+// to enable tooltips with the default configuration
```
[the remark this answers is not in the document]
Comment (XhmikosR) [reply site/content/docs/5.2/getting-started/javascript.md:304]: I still feel this is more correct as is.

File: site/content/docs/5.2/getting-started/javascript.md [hunk 2]
```diff
@@ -302,11 +301,17 @@ const tooltip = new bootstrap.Tooltip(yourTooltipEl, {
...
+$('[data-bs-toggle="tooltip"]').tooltip();
```
[the remark this answers is not in the document]
Comment (XhmikosR) [reply site/content/docs/5.2/getting-started/javascript.md]: Fixed. BTW shouldn't eslint complain about the extra semicolons?

[the remark this answers is not in the document]
Comment (julien-deramond) [reply site/content/docs/5.2/getting-started/javascript.md]: 🤔 eslint doesn't seem to analyze our Markdown files. If I add a `console.log()` in this "```js" there's no error until I change that:

```diff
diff --git a/package.json b/package.json
index 736b7a21d..12e19c952 100644
--- a/package.json
+++ b/package.json
@@ -63,7 +63,7 @@
     "js-compile-standalone-esm": "rollup --environment ESM:true,BUNDLE:false --config build/rollup.config.js --sourcemap",
     "js-compile-bundle": "rollup --environment BUNDLE:true --config build/rollup.config.js --sourcemap",
     "js-compile-plugins": "node build/build-plugins.js",
-    "js-lint": "eslint --cache --cache-location .cache/.eslintcache --report-unused-disable-directives --ext .html,.js .",
+    "js-lint": "eslint --cache --cache-location .cache/.eslintcache --report-unused-disable-directives --ext .html,.js,.md .",
     "js-minify": "npm-run-all --aggregate-output --parallel js-minify-*",
     "js-minify-standalone": "terser --compress passes=2 --mangle --comments \"/^!/\" --source-map \"content=dist/js/bootstrap.js.map,includeSources,url=bootstrap.min.js.map\" --output dist/js/bootstrap.min.js dist/js/bootstrap.js",
     "js-minify-standalone-esm": "terser --compress passes=2 --mangle --comments \"/^!/\" --source-map \"content=dist/js/bootstrap.esm.js.map,includeSources,url=bootstrap.esm.min.js.map\" --output dist/js/bootstrap.esm.min.js dist/js/bootstrap.esm.js",
```

But if it is the correct way to enable it, there will be more modifications to do.

**Review by julien-deramond — APPROVED** [review]
Modifications LGTM. Things related to eslint are interesting and could be tackled in another PR of course.

File: site/content/docs/5.2/getting-started/javascript.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (XhmikosR) [reply site/content/docs/5.2/getting-started/javascript.md]: Yeah, the right way I think it's the same way we added HTML there. No hurry, but maybe it's worth having markdown files lint too.

**Pull request merged**